index.js

```diff
@@ -0,0 +1,82 @@
+$(document).ready(() => {
+  const loadTrips = function loadTrips() {
+    $.get('https://trektravel.herokuapp.com/trips/', (response) => {
+      console.log('success!');
+
+      response.forEach((trip) => {
+        const tripInfo = $(`<article class="article-block" data-id=${trip.id} data-name=${trip.name} data-continent=${trip.continent}>
+        <p>${trip.name}</p>
+        <ul class="details"></ul>
+        <button class="trip-details">View Trip Details</button>
+        </article>`);
+
+        $('#trips').append(tripInfo);
+
+        // moving this inside of the first each loop so that we have access to the ID so we can pass it to the loadTripDetails function.
+        tripInfo.find('.trip-details').on('click', () => {
+          console.log('WORKING?????');
+          loadTripDetails(trip.id);
+        });
+
+        const loadTripDetails = function loadTripDetails(id) {
+          $.get(`https://trektravel.herokuapp.com/trips/${id}`, (response) => {
```

Comment:
You've already used `response` for a variable name in this scope, maybe name this more specifically instead, maybe `loadTripDetailsResponse`.